mods/classic-theme-enable-with-extended-compatibility.wh.cpp

```diff
@@ -0,0 +1,656 @@
+
+* [Aero Tray](https://windhawk.net/mods/aero-tray)
+* [Classic Conhost](https://windhawk.net/mods/classic-conhost)
+* [Classic Task Dialog Fix](https://windhawk.net/mods/classic-taskdlg-fix)
+* [Classic Taskbar 3D buttons with extended compatibility](https://windhawk.net/mods/classic-taskbar-buttons-lite-vs-without-spacing) or alternatively, [Classic Taskbar 3D buttons Lite](https://windhawk.net/mods/classic-taskbar-buttons-lite)
+* [Classic Taskbar background fix](https://windhawk.net/mods/classic-taskbar-background-fix) (simple to install) or alternatively, [@valinet's modified version of OpenShell StartMenuDLL.dll](https://github.com/valinet/ExplorerPatcher/discussions/167#discussioncomment-1616107) (requires more manual work to install)
+* [Classic theme transparency fix](https://windhawk.net/mods/classic-theme-transparency-fix)
+* [Classic UWP Fix](https://windhawk.net/mods/classic-uwp-fix)
+* [Clientedge Everywhere](https://windhawk.net/mods/clientedge-in-apps)
+* [Disable rounded corners in Windows 11](https://windhawk.net/mods/disable-rounded-corners) (Win 11 only)
+* [DWM Ghost Mods](https://windhawk.net/mods/dwm-ghost-mods)
+* [Fix browsers for Windows Classic theme](https://windhawk.net/mods/classic-browser-fix)
+* [Fix Classic Theme Maximized Windows](https://windhawk.net/mods/classic-maximized-windows-fix)
+* [Win32 Tray Clock Experience](https://windhawk.net/mods/win32-tray-clock-experience) (Win 10 only)
+
+There are other classic theme related mods in Windhawk which I did not list here for one or other reason. Your experience and preferences may differ. After getting set up with above and feeling like exploring more, you may want to try the other mods out.
+
+After installing the above mentioned mods, your Windhawk window should look similar to the following screenshot. Of course you may have other mods installed from before as well.
+
+![A Windhawk screenshot with classic theme mods](https://raw.githubusercontent.com/levitation-opensource/my-windhawk-mods/main/screenshots/classic-theme-mods-in-windhawk.png)
+
+
+
+## 6. How to configure system start of Windhawk
+
+Starting Windhawk early improves the probability that classic theme is enabled by the time Taskbar process is launched during login. 
+
+*In contrast, when Windhawk is activated normally then there is increased chance that the Taskbar process starts before classic theme is enabled - then the Taskbar would not have classic appearance and the user needs to restart the Taskbar manually later in order to apply classic theme to Taskbar.*
+
+Steps to enable system start of Windhawk:
+
+1. Start Task Scheduler
+2. Open "Task Scheduler Library" section 
+3. Find the row titled "WindhawkRunUITask", open it by double clicking
+4. Go to Triggers
+5. Click "New..." button
+6. Select Begin the task: "At startup"
+7. OK
+8. Click "New..." button
+9. Select Begin the task: "At log on"
+10. OK
+11. OK
```

Comment:
Does it actually help? It just launches the UI, unless I'm missing something.

Reply:
@m417z Maybe you are correct, I am not sure about this part, and likely you know it better. It seemed that creating a system start trigger improved the chance that Windhawk is started early enough. I see that Windhawk is also run as a service. Could there be any chance that the service does not hook into other processes until UI process is launched?

Reply:
The two are mostly independent, and the service doesn't wait for the UI. By default, the UI doesn't run on startup, the service does. In fact, the UI folder can be fully removed after installing mods and Windhawk will keep on working.

Reply:
Thank you for the idea, sounds good! I will try it.

Reply:
I imagine it should be a system service which is non-essential. Else there would be a risk that if Windhawk is unable to start for some reason, then the system service does not start either.

Reply:
It'd be better, but perhaps you must choose an essential service to ensure the load time that's needed for this mod. Hooking into `winlogon.exe` already has risks and might require booting in safe mode to disable Windhawk if something goes wrong.

Reply:
Alternative would be perhaps creating a Task Scheduler task that triggers Windhawk service start using net start command.

Reply:
You can try that, but it feels more fragile to me. Also, I don't know how early the Task Scheduler service runs, perhaps not earlier than Windhawk does now.